### DISCUSSION TITLE
Make esc instantly pause no matter the setting of Hold-to-confirm activiation time

### DISCUSSION THREAD

**Discussion by GhostylGit** [ppy/osu#15256] [Ideas] [answered]
Currently pausing and quick restarting the map are both bound to this setting. The problem I have with this is being able to instantly pause during gameplay is very useful, especially if a break is very short however this makes the quick restart keybind trigger instantly aswell making it possible to fat finger or accidentally press it because of developed muscle memory when restarting a map. This is how this functions in stable currently.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/6397
